MainWindow.cpp

```diff
@@ -29,10 +29,12 @@
 
 #include <functional>
 #include <unordered_map>
+#include <QtGlobal>

```

Comment:
Yeah, no, don't include this one if you have to. We already came up with a general not to do this until you've hit like 10 or 15 Qt includes. Only at that point or when you absolutely have to do we allow including this monolithic header.